src/ui/actions/timeline.ts

```diff
@@ -245,15 +249,15 @@ export function seekToTime(targetTime: number): UIThunkAction {
     }
 
     const nearestEvent = mostRecentPaintOrMouseEvent(targetTime) || { point: "", time: Infinity };
-    const pointNearTime = await ThreadFront.getPointNearTime(targetTime);
-
-    if (Math.abs(pointNearTime.time - targetTime) > Math.abs(nearestEvent.time - targetTime)) {
-      dispatch(seek(nearestEvent.point, targetTime, false));
-    } else {
-      // I would prefer that we also use pointNearTime.time here, for accuracy,
-      // but it would be super annoying when it is off. Maybe when we have a
-      // more exact method.
-      dispatch(seek(pointNearTime.point, targetTime, false));
+    let bestPoint = nearestEvent;
+    try {
+      const pointNearTime = await ThreadFront.getPointNearTime(targetTime);
+      if (Math.abs(pointNearTime.time - targetTime) < Math.abs(nearestEvent.time - targetTime)) {
+        bestPoint = pointNearTime;
+      }
+    } catch (e) {
+    } finally {
```

Comment:
I think `finally` is only needed if you re-throw the error, but it's also fine either way 😄